exchanges/bitfinex/bitfinex.go

```diff
@@ -1688,7 +1688,7 @@ func (b *Bitfinex) CancelMultipleOrdersV2(ctx context.Context, orderID, clientOr
 					cancelledOrder.AuxLimitPrice = f
 				}
 			}
-			cancelledOrders[y] = cancelledOrder
+			cancelledOrders = append(cancelledOrders, cancelledOrder)
```

Comment:
Does this fix something?
In any case should be in it's own commit, and if it's a fix PR'd separately.
If it's fixing something, should have a test to prove it was broken before too 😄